district_data_import.py

```diff
@@ -144,6 +146,53 @@
             'ZIP': district_record['zip']
         })
 
+        # Update for potential switch of parent name order
+
+        curr_informal_sal = dp_donorrecord['INFORMAL_SAL']
+        curr_main_f_name = dp_donorrecord['FIRST_NAME']
+        curr_spouse_f_name = dp_donorrecord['SP_FNAME']
+
+        dp_donorrecord_with_pnames = district_data_utils.create_dp_donorrecord(district_record=district_record, school_year=args.school_year)
```

Comment:
Can we change this from `dp_donorrecord_with_pnames` to something more accurate like `generated_dp_donnorrecord`?

Reply:
I think the name is why I had such a hard time understanding this part of the change.

Reply:
The previous use of this method creates a variable called dp_donorrecord_for_matching. I'm going to call this one dp_donorrecord_for_update to keep things consistent. Calling it generated_dp_... doesn't really say much given that it is the output of a method called create_dp_... anyway. The problem I've had is that the convention to use dp_donorrecord as a long suffix doesn't actually convey the key point that record is created using district data. A better convention may be to use dp_rec_from_dist_data as the suffix every time we use this create function...

Reply:
I don't care too much what the name is, I just found `dp_donorrecord_with_pnames` to be very confusing.